### ISSUE TITLE
Emojis show above the name of the user, making reading a little uncomfortable

### ISSUE THREAD

**Issue by gmiz** [qTox/qTox#6135]
##### Brief Description

OS: Linux (Fedora 24)
qTox version: v1.17.2
Commit hash:  465d82606813439ee14425f7462903bb39f2da30
toxcore:  0.2.11
Qt:  5.7.1

Reproducible: Always

##### Steps to reproduce

- open qtox
- select the emojione emoji pack through settings
- set the emoji size to 50px
- get into any chat
- send an emoji

##### Observed Behavior

The emoji is kinda printed above the username, giving a bit of a weird effect.

##### Expected Behavior

The emoji should be printed from top to bottom, just to make it clear who sent the message. This is not a bug, but just a way to make it more easy to read messages.

##### Additional Info

See how when I first send an emoji, it shows printed above quite a lot, but when I just type text, it works fine:
![tmp](https://user-images.githubusercontent.com/47925487/81411319-7f6de900-9142-11ea-9036-3164774dda1d.png)

I made this in GIMP, to show what I think would be much better:
![tmp](https://user-images.githubusercontent.com/47925487/81411933-59951400-9143-11ea-9ff5-047c3d61a2d4.png)

I think it's much easier to read like that.

**Comment by sudden6** [comment]
I agree, the alignment here doesn't really look right, but I think rather than moving the name, the vertical alignment of the emojis should be fixed, such that they are centered with the rest of the text.

**Comment by gmiz** [comment]
@sudden6 yes, here's an example from discord, which I think is what we should be going for:
![tmp](https://user-images.githubusercontent.com/47925487/81473301-c79e1180-91fd-11ea-9471-d7efa98fbd0f.png)
Also, see how line spacing there doesn't change. That's kinda cool too.
Here's how it looks in qtox, even when the emojis are set to something as low as 20px and the text is 16px:
![tmp](https://user-images.githubusercontent.com/47925487/81473235-5c543f80-91fd-11ea-8d04-d11716681bf6.png)
So I think the emojis should be rendered kinda separately in a way, so that they don't ruin the line spacing, even though that's not too important. It's a minor problem.

Do we need a separate issue for the line spacing?

**Comment by sudden6** [comment]
No, I think this is connected somehow.
